tests/Tests/400__ods_dashboard/400__ods_dashboard.robot

```diff
@@ -271,3 +292,27 @@ Filter By Using More Than One Filter And Check Output
     FOR    ${id}    IN    @{LIST_OF_IDS_FOR_COMBINATIONS}
         Deselect Checkbox Using Id    id=${id}
     END
+
+Delete Multiple Builds
```

Comment:
This could be a keyword in tests/Resources/Page/OCPDashboard/Builds/Builds.robot, having  namespace and build_name_list as parameters

Reply:
Updated.